src/measurement_kit/cmd/whatsapp.cpp

```diff
@@ -0,0 +1,33 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "../cmdline.hpp"
+
+namespace whatsapp {
+
+#define USAGE                                       \
+    "usage: measurement_kit [options] whatsapp\n"   \
+
+int main(std::list<Callback<BaseTest &>> &initializers, int argc, char **argv) {
+    mk::nettests::WhatsappTest test;
+    int ch;
+    while ((ch = getopt(argc, argv, "B:f:")) != -1) {
```

Comment:
Uhm, here you actually support `-B <arg>` and `-f <arg>`. You should probably empty the format string.

Reply:
acked